gulpfile.babel.js

```diff
@@ -1,4 +1,5 @@
-import gutil from 'gulp-util'
+const log = require('fancy-log')
```

Comment:
Can we use `@wdio/logger` for consistency?

Reply:
Done